### PR TITLE
Apply shellcheck

### PR DESCRIPTION
**Pull request by SergiiDmytruk**
This is in preparation of updating scripts in #42 to start with `shellcheck` being silent.

### CODE REVIEW

File: 90anti-evil-maid/anti-evil-maid-unseal [hunk 1]
```diff
@@ -130,10 +131,10 @@ fi
 # check freshness token
 
 log "Unsealing freshness token..."
-if tpm_unsealdata $Z -i "$SEALED_SECRET_FRE" -o "$UNSEALED_SECRET" \
+if tpm_unsealdata "$Z" -i "$SEALED_SECRET_FRE" -o "$UNSEALED_SECRET" \
```
Comment (marmarek) [inline 90anti-evil-maid/anti-evil-maid-unseal]: This one (and few similar below) was intentionally not quoted - if no `-z` is intended, it should give no argument, not empty argument.

**Review by marmarek — CHANGES_REQUESTED** [review]
Thank you for splitting it into separate commits, it made the review much easier.

Besides the comment below, have you tested if adding `set -u` didn't introduced any issue?

File: 90anti-evil-maid/anti-evil-maid-unseal (hunk 1 above)
Comment (SergiiDmytruk) [reply 90anti-evil-maid/anti-evil-maid-unseal]: I thought it might be, undone changes with `$Z` and suppressed warnings.

**Comment by SergiiDmytruk** [comment]
> have you tested if adding `set -u` didn't introduced any issue?

Not really.  I decided to add it because `shellcheck` should catch such errors (it did in 2a32b3ca86cd62674d28552764694d391dae6902), but on the second thought it's probably safer to move the commit to #42 and see if it will cause any issues while testing new TPM2 support and verifying that TPM1 still works fine.  Removed the change from this PR.

**Review by marmarek — APPROVED** [review]

**Pull request merged**